openscap_daemon/cve_scanner/cve_scanner.py

```diff
@@ -354,15 +363,23 @@ def get_cid(self, input_name):
                     break
         return None
 
+    def parse_image_name(self, input_name):
+        """
+        Parse image name and return its parts as tuple
+        :param input_name:
+        :return: (name,tag)
+        """
+        m = self.name_regex.match(input_name)
+        if m:
+            return (m.group(1), m.group(2))
+        else:
+            return None
+
     def _namesearch(self, input_name):
         """
         Looks to see if the input name is the name of a image
         """
-        if ":" in input_name:
-            image_name, tag = input_name.split(":")
-        else:
-            image_name = input_name
-            tag = None
+        image_name, tag = self.parse_image_name(input_name)
```

Comment:
What if the `self.parse_image_name()` method returns `None`? Should it be checked here?